README.md

```diff
@@ -1,9 +1,156 @@
 # User Login.
-This library is for user login with active directory account.
+This package provides user login functionality.
 
 Built for [Laravel 11](https://laravel.com/).
 
 ## Installation
 
-Add the library via Composer: `composer require networkrailbusinesssystems/user-login`
+Add the library using Composer:
+```
+composer require networkrailbusinesssystems/user-login
+```
+The service provider will be automatically registered.
 
+## Configuration
+
+#### 1. Export the config file
+To customize the package, export the configuration file using Artisan:
+
+```
+php artisan vendor:publish --provider="NetworkRailBusinessSystems\UserLogin\Providers\UserLoginServiceProvider" --tag=config
+```
+
+#### 2. Publish the views.
+To modify the login views, publish the package views using Artisan:
+
+```
+php artisan vendor:publish --provider="NetworkRailBusinessSystems\UserLogin\Providers\UserLoginServiceProvider" --tag=views
+```
+
+#### 3. Available Configuration Options
+After publishing, you can modify the following configuration options in the config/user-login.php file:
+
+```
+return [
```

Comment:
As per the traits, just point them to the config file, or provide a table of options as per https://github.com/Network-Rail-Business-Systems/support-page/?tab=readme-ov-file#configuration

Reply:
fixed